### PR TITLE
Update to libGDX 1.11.0

### PR DESCRIPTION
**Pull request by Dgzt**

### CODE REVIEW

File: ui/CHANGES.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-#### Version: 1.5.1-SNAPSHOT (libGDX 1.10.0)
+#### Version: 1.5.1-SNAPSHOT (libGDX 1.11.0)
+
+- Updated to libGDX 1.11.0
+- - **Changed**: The project now uses Java 1.8 compatibility level, required by LWJGL3
```
Comment (czyzby) [inline ui/CHANGES.md:4]: LWJGL3 will work with a 1.7-compatible library. On the other hand, I'm not sure if iOS backend fully supports Java 8 standard library.

Comment (Dgzt) [reply ui/CHANGES.md]: You are right. The lwjgl3 is only used in tests, so enough to set the test's target compatibility level to 1.8.

**Comment by Frosty-J** [comment]
gdx-setup 1.11.0 uses Gradle 7. Be sure to change `compile` in `README.md`, as this no longer works.

**Comment by czyzby** [comment]
Thanks @Frosty-J.

**Pull request merged**